installer/conf/omsagent.d/patch_management.conf

```diff
@@ -39,13 +39,31 @@
     log_level info
 </source>
 
-<filter oms.update_progress>
+<filter oms.apt_update_progress>
     @type grep
     regexp1 apt-logs Requested
     exclude1 apt-logs Purge|Remove
 </filter>
 
-<filter oms.update_progress>
+<filter oms.apt_update_progress>
+    @type filter_linux_update_run_progress
+    current_update_run_file "/var/opt/microsoft/omsagent/state/schedule_run.id"
+    log_level info
+</filter>
+
+<source>
+    @type sudo_tail
+    tag oms.yum_update_progress
+    path /var/log/yum.log
+    format /(?<update-action-date>.*\d{2}:\d{2}:\d{2}) (?<update-action>.*):(?<package-name>.*)/
+    run_interval 60s
+    pos_file /var/opt/microsoft/omsagent/state/yum.log.pos
+    keep_time_key true
```

Comment:
Does keep_time_key work? Since you didn't specify the time_key, it uses the default one "time". While your regex doesn't have such extraction.